Formula/brainfuck.rb

```diff
@@ -0,0 +1,30 @@
+class Brainfuck < Formula
+  desc "Interpreter for brainfuck written in C"
+  homepage "https://github.com/fabianishere/brainfuck"
+  url "https://github.com/fabianishere/brainfuck/archive/2.7.1.tar.gz"
+  sha256 "06534de715dbc614f08407000c2ec6d497770069a2d7c84defd421b137313d71"
+
+  head "https://github.com/fabianishere/brainfuck.git"
+
+  option "with-debug", "Compile interpreter with debug support"
+  option "with-examples", "Install brainfuck example programs"
+  option "with-static-lib", "Build a static library"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = std_cmake_args
+    args << "-DBUILD_SHARED_LIB=ON"
+    args << "-DENABLE_DEBUG=ON" if build.with? "debug"
+    args << "-DINSTALL_EXAMPLES=ON" if build.with? "examples"
+    args << "-DBUILD_STATIC_LIB=OFF" if build.without? "static-lib"
+
+    system "cmake", ".", *args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/brainfuck", "-e", "++++++++[>++++++++<-]>+.+.+."
```

Comment:
This needs to verify the functionality of the software not just check for a zero exit code. Take a look at `jose` and `vim`.

Reply:
Is the new test better?